light/detector_test.go

```diff
@@ -17,8 +17,6 @@ import (
 )
 
 func TestLightClientAttackEvidence_Lunatic(t *testing.T) {
-	t.Skip("Voter selection in Ostracon only supports sequential verification mode, " +
-		"but Ostracon has a few test case for skipping mode.")
```

Comment:
Why do some test cases that were currently skipped now work in v0.34.9?

Reply:
I've just prepared the test data corresponding to the sequential mode. 

- https://github.com/line/ostracon/pull/349/commits/53712994b4795f85bac00187bf66937884dc86d9
  - First of all, `keys.GenSignedHeader` doesn't support sequential mode, so I used `keys.GenSignedHeader` and `keys.GenSignedHeaderLastBlockID ` via `genMockNodeWithKey`
- https://github.com/line/ostracon/pull/349/commits/e4bea390b3b92d78a0e41405dac8e89a5b0cbe35#diff-60ae289a89b3a623764d2927fdfc1d16e11525487af6e088e3e60ab98e57a025R40-R51
  - Should prepare previous divergence height, keys, validators, and voters correctly
- https://github.com/line/ostracon/pull/349/commits/68beef483e77f1335615b24e87aec0f6cdba5741#diff-60ae289a89b3a623764d2927fdfc1d16e11525487af6e088e3e60ab98e57a025R148-R156
  - Should use `keys.GenSignedHeaderLastBlockID`
- https://github.com/line/ostracon/pull/349/commits/d23562afde0a01c9ff62d081d095cc5de044d360#diff-60ae289a89b3a623764d2927fdfc1d16e11525487af6e088e3e60ab98e57a025R233-R245
  - Should prepare forged keys and etc.

Reply:
OK, you've fixed it to work.